src/pages/scripts/tutorials/script-result/index.md

```diff
@@ -1,50 +1,49 @@
 # Setting Script Result
 
-InDesign now has the functionality to "set result" of a UXP script. After execution, a script can have its own result which then can be used for debugging purposes on InDesign Server. The result of the script can also be stored in a variable and be used by another script in both InDesign App and InDesign Server. Setting the script result can be accomplished using the script.`setResult()` API, the sections below demonstrate the various use cases. 
```

Comment:
This tutorial appears to be applicable to both - InDesign Server and InDesign app. Yet, the tutorial is written in a fashion that focuses on IDS. Use other tutorials as an example and update its content. @anoopvalomkot

Reply:
ok, I have made some corrections below.